ggj25/scripts/bubball.gd

```diff
@@ -76,14 +55,17 @@ func get_winning_team() -> int:
 	else:
 		return 0
 
+
 func _on_left_goal_bubble_collected(team_id: int) -> void:
 	if started:
+		hud.show_message("RIGHT TEAM SCORES!")
```

Comment:
Is the team naming consistent now? We had left and right vs one and two before, but are we now always left and right?

Reply:
Hmmm I will double check, I think the end of game naming is still different